benchmarks/spinning/go/server.go

```diff
@@ -0,0 +1,89 @@
+// MIT License
+
+// Copyright (c) 2022 EASE lab
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+// Package main implements a server for Aes service.
+package main
+
+import (
+	"context"
+	"flag"
+	"net"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	pb "github.com/vhive-serverless/vSwarm-proto/proto/aes"
+	tracing "github.com/vhive-serverless/vSwarm/utils/tracing/go"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/reflection"
+)
+
+var (
+	zipkin  = flag.String("zipkin", "http://localhost:9411/api/v2/spans", "zipkin url")
+	address = flag.String("addr", "0.0.0.0:50051", "Address:Port the grpc server is listening to")
+)
+
+// server is used to implement aes.AesServer.
+type server struct {
+	pb.UnimplementedAesServer
+}
+
+// ShowEncryption implements aes.AesServer
+func (s *server) ShowEncryption(ctx context.Context, in *pb.PlainTextMessage) (*pb.ReturnEncryptionInfo, error) {
+	startTime := time.Now()
+	for i := 0; i < 30000000; i++ {
```

Comment:
Hi dhschall, first of all thank you so much for the review!! My apologies for these mistakes. I have no permission to push to vhiveeasse docker so i used my local, will asked for permission later. For these benchmarks i actually wanted to fix the iterations (instead of fixing it to spin for xx milliseconds) in order to check how latency changes when cpu frequency changes as part of my project to optimise power usage (tune down frequency for tasks like I/O bounded to save power since latency would not be affected by cpu frequency regardless) so we probably should not combine these 2 benchmarks together and make it run for a fixed amount of time, right? what do u think?